deploy/examples/crds.yaml

```diff
@@ -7331,11 +7331,7 @@ spec:
                     endpoint:
                       pattern: ^https*://
                       type: string
-                  required:
-                    - endpoint
```

Comment:
Does the documentation need to be updated in some way with these fields not being required anymore?

Reply:
The docs actually already mention it's optional.